Blockzilla.xcodeproj/xcshareddata/xcschemes/Focus (Enterprise).xcscheme

```diff
@@ -37,6 +37,10 @@
                BlueprintName = "XCUITest"
                ReferencedContainer = "container:Blockzilla.xcodeproj">
             </BuildableReference>
+            <LocationScenarioReference
+               identifier = "London, England"
+               referenceType = "1">
+            </LocationScenarioReference>
```

Comment:
oh wait, sorry, I set this for google webpage to show the current location in testGoogleLocation, but apparently it is buggy- (it simulates to South Africa instead of England) so there's no need for this.